openforcefield/tests/test_toolkits.py

```diff
@@ -657,77 +659,201 @@ def test_generate_conformers(self):
+                                     toolkit_registry=toolkit_registry)
 
+        # Try again, with strict_n_confs as true, but not including use_confs, so the
+        # recommended number of confs will be generated
+        molecule.compute_partial_charges_am1bcc(toolkit_registry=toolkit_registry,
+                                                strict_n_conformers=True)
 
-        with pytest.raises(NotImplementedError) as excinfo:
-            charge_model = 'notARealChargeModel'
-            molecule.compute_partial_charges(toolkit_registry=toolkit_wrapper)#, charge_model=charge_model)
-
-        # TODO: Test all supported charge models
-        # TODO: "amber" and "amberff94" only work for a subset of residue types, so we'll need to find testing data for
-        # those
-        # charge_model = [,'amber','amberff94']
-        # The 'noop' charge model doesn't add up to the formal charge, so we shouldn't test it
-        # charge_model = ['noop']
-        for charge_model in ['mmff', 'mmff94', 'am1bcc', 'am1bccnosymspt', 'am1bccelf10']:
-            with pytest.raises(NotImplementedError) as excinfo:
-                molecule.compute_partial_charges(toolkit_registry=toolkit_wrapper) #, charge_model=charge_model)
-                charge_sum = 0 * unit.elementary_charge
-                for pc in molecule._partial_charges:
-                    charge_sum += pc
-                assert 0.999 * unit.elementary_charge < charge_sum < 1.001 * unit.elementary_charge
-        # For now, I'm just testing AM1-BCC (will test more when the SMIRNOFF spec for other charges is finalized)
-        molecule.compute_partial_charges_am1bcc(toolkit_registry=toolkit_wrapper)
-        charge_sum = 0 * unit.elementary_charge
-        for pc in molecule._partial_charges:
+
+
+    @pytest.mark.skipif(not OpenEyeToolkitWrapper.is_available(), reason='OpenEye Toolkit not available')
+    @pytest.mark.parametrize("partial_charge_method", ['am1bcc', 'am1-mulliken', 'gasteiger'])
+    def test_assign_partial_charges_neutral(self, partial_charge_method):
+        """Test OpenEyeToolkitWrapper assign_partial_charges()"""
+        from openforcefield.tests.test_forcefield import create_ethanol
+        toolkit_registry = ToolkitRegistry(toolkit_precedence=[OpenEyeToolkitWrapper])
+        molecule = create_ethanol()
+        molecule.assign_partial_charges(toolkit_registry=toolkit_registry,
+                                        partial_charge_method=partial_charge_method)
+        charge_sum = 0. * unit.elementary_charge
+        for pc in molecule.partial_charges:
             charge_sum += pc
-        assert 0.999 * unit.elementary_charge < charge_sum < 1.001 * unit.elementary_charge
+        assert -1.e-5 < charge_sum.value_in_unit(unit.elementary_charge) < 1.e-5
+
+    @pytest.mark.skipif(not OpenEyeToolkitWrapper.is_available(), reason='OpenEye Toolkit not available')
+    @pytest.mark.parametrize("partial_charge_method", ['am1bcc', 'am1-mulliken'])
```

Comment:
Yes -- Gasteiger isn't conformer-dependent, so this test (ensuring that charge methods generate different charges for different conformations) doesn't apply to it. Added
`Skip Gasteiger because it isn't conformer dependent.`
to the test docstring